### ISSUE TITLE
Can you add Veldhoven (Afvalinfo.nl)

### ISSUE THREAD

**Issue by Rikkie80** [pippyn/Home-Assistant-Sensor-Afvalbeheer#58]
I live in Veldhoven (postal codes 5500 - 5509) and the calendar is generated by Afvalinfo.nl.
the api used by afvalinfo.nl seems to be almost similar to the ones you integrated already. Can this one be integrated as well (is available in the Afvalinfo integration, but I like your integration more)?

**Comment by pippyn** [comment]
Will look into it!

**Comment by pippyn** [comment]
Can you give an postcode and house number tot test with?

**Comment by Rikkie80** [comment]
5503PN 15

**Comment by basschipper** [comment]
I had a look at this as well, but I can't find an api at all? The only useful thing I see here is the ical thing. In the ical thing there is a reference to the company that created afvalinfo.nl, maybe send them an email if an api is available?

**Comment by pippyn** [comment]
I couldn't find a API request either. And implementations for other projects use a scraper. Which is not an option for this sensor.

**Comment by Rikkie80** [comment]
I sent a mail to afvalinfo.nl with the request for an API. Let's wait for the answer. Thanks for the effort thus far

**Comment by Rikkie80** [comment]
Thanks for the integration. It worked for Veldhoven for a long time, but now it is broken again... Can you please have a look?

**Comment by De-Klump** [comment]
Is there any news on location Veldhoven?

**Comment by xirixiz** [comment]
@pippyn It's quiet easy to add Veldhoven maybe. I  didn't check your code for this, but I have this collector: https://github.com/xirixiz/homeassistant-afvalwijzer/blob/master/custom_components/afvalwijzer/collector/icalendar.py

Maybe you can do some good old copy paste work to make it work for your integration.

**Comment by pippyn** [comment]
Thanks @xirixiz, but if I read your code correctly it's more of a ics/ical scraper. Right?
I'm not really a fan of that solution to be honest. I would rather use an API.

**Comment by xirixiz** [comment]
Yeah correct. I don't prefer that as well, but since someone added this for me, I was oke with it (not much effort). It's running for quiet some time already. If it breaks often or I need to make exotic solution for other "gemeenten", then I`ll remove it as well. Anyway, so far, so good.
